lib/results.js

```diff
@@ -139,8 +144,8 @@ function encodeResult(res, count) {
     output += (res.ok ? 'ok ' : 'not ok ') + count;
     output += res.name ? ' ' + res.name.toString().replace(/\s+/g, ' ') : '';
 
-    if (res.skip) output += ' # SKIP';
-    else if (res.todo) output += ' # TODO';
+    if (res.skip) output += ' # SKIP' + (typeof res.skip === 'string' ? ' ' + res.skip : '');
```

Comment:
Sorry this change crept in, It is unintentional regarding this PR. Sorry.

Reply:
let’s revert it then?

Reply:
https://github.com/substack/tape/pull/473#discussion_r296451500